client/app/screens/InviteList.js

```diff
@@ -83,7 +84,16 @@ export default class InviteListScreen extends React.Component {
           });
       });
 
-    // Initial load of invites
+    let reloadInvites = function(invites) {
+      this.setState({ dataSource: ds.cloneWithRows(invites) });
+    }.bind(this);
+
+    DeviceEventEmitter.addListener('onDefaultMessage', function(event) {
```

Comment:
So there is RCTEventEmitter in iOS which looks like the "golden scenario" but similar APIs don't exist in Android.  I dug into various other event emitter APIs but they all seem to have a similar "what docs" problem